app/presenters/organisations/show_presenter.rb

```diff
@@ -1,5 +1,7 @@
 module Organisations
   class ShowPresenter
+    include ActionView::Helpers
```

Comment:
Is this so you can use `content_tag`? You can pull in the bit that is specifically needed, e.g: `include ActionView::Helpers::TagHelper`